crates/apollo-compiler/src/lib.rs

```diff
@@ -18,6 +18,9 @@ pub struct ApolloCompiler {
     pub db: RootDatabase,
 }
 
+/// A read-only, `Sync` snapshot of the database.
+type Snapshot = salsa::Snapshot<RootDatabase>;
```

Comment:
should this be pub then?
```suggestion
pub type Snapshot = salsa::Snapshot<RootDatabase>;
```

Reply:
🤦 

thanks!